packages/theme/src/cli/utilities/theme-environment/hot-reload/server.ts

```diff
@@ -287,7 +293,7 @@ function hotReloadSections(key: string, ctx: DevServerContext) {
     const content = ctx.localThemeFileSystem.files.get(key)?.value
     if (content) {
       const sections: SectionGroup | undefined = parseJSON(content, null)?.sections
-      for (const sectionName of Object.keys(sections || {})) {
+      for (const sectionName of Object.keys(sections ?? {})) {
```

Comment:
@frandiox can you just doublecheck that these are safe?

I'm pretty confident but would love another set of eyes, since we are intentionally using `||` in [some places](https://github.com/Shopify/cli/pull/4674/files#diff-d857e2b583dcee2197fc538eb4d35aabb5ce83cee18ebb970ece7854dda8ba07R139-R140)

Reply:
I think it should be OK. Even if the JSON is malformed and `sections` is an empty string or similar it should still work 😅